script.js

```diff
@@ -4,3 +4,112 @@
 // - A User should only be able to select one box per turn
 // - A User should not be able to select a box that is already selected
 // - When game is over (whether a win or tie) the users should be prompted to play again and the game should be reset
+
+let gamePiece = "";
+let playerTurn = true;
+let text = document.getElementById("text");
+const xGamePiece = "X";
+const oGamePiece = "O";
+const one = document.getElementById("1");
+const two = document.getElementById("2");
+const three = document.getElementById("3");
+const four = document.getElementById("4");
+const five = document.getElementById("5");
+const six = document.getElementById("6");
+const seven = document.getElementById("7");
+const eight = document.getElementById("8");
+const nine = document.getElementById("9");
+const gameSpaces = document.querySelectorAll(".gameSpace");
+
+gameSpaces.forEach(addEventListenerToGameSpaces);
+
+text.innerHTML += "Player X turn";
```

Comment:
you can probably move this down under the `addEventListenerToGameSpaces` function to kind of clean this up logically a bit